go/libraries/doltcore/sqle/sqlddl_test.go

```diff
@@ -1511,7 +1511,7 @@ INSERT INTO child_non_unq VALUES ('1', 1), ('2', NULL), ('3', 3), ('4', 3), ('5'
 	require.NoError(t, err)
 	_, err = ExecuteSql(dEnv, root, "INSERT INTO child_unq VALUES ('6', 5)")
 	if assert.Error(t, err) {
-		assert.Contains(t, err.Error(), "UNIQUE constraint violation")
+		assert.Contains(t, err.Error(), "duplicate unique key")
```

Comment:
Can this be something like

    assert.True(errors.Is(err, ErrDuplicateKey))

So this doesn't fail if the str changes